comit/src/network/swap_digest.rs

```diff
@@ -88,6 +88,10 @@ pub struct HbitHerc20 {
     pub token_contract: identity::Ethereum,
 }
 
+/// A new-type that allows us to implemented `ToDigestInput` on foreign types.
+#[derive(Clone, Debug, PartialEq)]
+pub struct Digestable<T>(pub T);
```

Comment:
Does the inner need to be pub? Maybe better to implement `From`.

Reply:
This struct doesn't enforce any invariants among its inner field so I don't see a reason why it should be private. Adding a `From` impl is just more code for no benefit IMO.